tests/algorithms.test.ts

```diff
@@ -1,6 +1,113 @@
-import { MADe, M_Estimator, Q, Hampel, A_Algorithm } from '../src/lib';
+import {
+  MADe,
+  M_Estimator,
+  Q,
+  Hampel,
+  A_Algorithm,
+  ReferenceValue,
+} from '../src/lib';
 
 describe('Algorithms', () => {
+  it('Reference Value (CASE1: x value is not range)', () => {
+    let xSample = 0.4;
+    const formulaeSample = [
```

Comment:
If this is the same sample data for all 3 methods, I suggest we move it outside of the function body so that we can simply reuse the same array.

Reply:
Good idea! 
However, x and formulae variable is different each case.